flytekit/sensor/file_sensor.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional, TypeVar
+
+from flytekit import FlyteContextManager
+from flytekit.sensor.base_sensor import BaseSensor
+
+T = TypeVar("T")
+
+
+class FileSensor(BaseSensor):
+    def __init__(self, name: str, config: Optional[T] = None, **kwargs):
```

Comment:
when would someone use the `config` field?

Reply:
timeout, token, etc.

Reply:
> don't really understand the need for both cloudpickle and jsonpickle. can you explain?

I use `jsonpickle` because pickle is not JSON serializable.  In translator, we use `json.dumps` to searlzie config, so I have to use `jsonpickle`.

https://github.com/flyteorg/flytekit/blob/36fe5813d83477f91056f61bd234e9dd232ec4a2/flytekit/models/task.py#L463